frontend/packages/dev-console/src/components/monitoring/__tests__/MonitoringPage.spec.tsx

```diff
@@ -32,6 +33,34 @@ describe('Monitoring Page ', () => {
           ns: 'test-proj',
         },
       },
+      canAccess: true,
+    };
+
+    window.SERVER_FLAGS.prometheusBaseURL = 'http://some-mock-url.com';
```

Comment:
This will impact other tests. You need to remember what the value is going in or maybe just set it to undefined.

If I modify your other test below to `canAccess: true`, moving the test around will change it to success or fail.

Reply:
Now setting value of the flag explicitly which makes sure that order of tests doesn't change the result. Also added another test for `canAccess=false && prometheusBaseURL='something'`.